loky/backend/_win_reduction.py

```diff
@@ -66,6 +64,7 @@ def rebuild_connection(reduced_handle, readable, writable):
     handle = reduced_handle.detach()
     return Connection(handle, readable=readable, writable=writable)
```

Comment:
Unrelated to this PR but we should add somme tests to cover this feature (picklability of Connection).